modules/apps/29-fee/keeper/msg_server_test.go

```diff
@@ -106,6 +108,15 @@ func (suite *KeeperTestSuite) TestPayPacketFee() {
 			},
 			true,
 		},
+		{
+			"refund account is module account",
+			func() {
+				msg.Signer = suite.chainA.GetSimApp().AccountKeeper.GetModuleAddress(disttypes.ModuleName).String()
```

Comment:
I think we can maybe remove this test case now?

Reply:
Is it worth keeping this in, and just change `disttypes.ModuleName` to `mock.ModuleName`?